hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/ReopenTableRegionsProcedure.java

```diff
@@ -224,6 +225,9 @@ protected void serializeStateData(ProcedureStateSerializer serializer) throws IO
     ReopenTableRegionsStateData.Builder builder = ReopenTableRegionsStateData.newBuilder()
       .setTableName(ProtobufUtil.toProtoTableName(tableName));
     regions.stream().map(ProtobufUtil::toRegionLocation).forEachOrdered(builder::addRegion);
+    if (CollectionUtils.isNotEmpty(regionNames)) {
```

Comment:
Since the regionsNames will not be null, we do not need test whether it is null or empty here? If it is empty, the below stream will add nothing.

Reply:
Let it be for backward compat?

Reply:
OK.